website/docs/getting-started/installation.md

```diff
@@ -50,9 +50,17 @@ docker run --rm ghcr.io/oss-review-toolkit/ort:latest [command]
 
 ## From binaries
 
-Preliminary binary artifacts for ORT are currently available via
-[JitPack](https://jitpack.io/#oss-review-toolkit/ort). Please note that due to limitations with the JitPack build
-environment, the reporter is not able to create the Web App report.
+A binary distribution of ORT can be downloaded from the
+[latest GitHub release](https://github.com/oss-review-toolkit/ort/releases/latest). The `ort-[version].zip` file
+contains binaries to run ORT in the `bin` folder:
+
+```shell
+# For Linux:
+bin/ort [command]
+
+# For Windows:
+bin\ort.bat [command]
+```
```

Comment:
Consider making this separate language-specific code blocks, like

For Linux:

```shell
bin/ort [command]
```

For Windows:

```batchfile
bin\ort.bat [command]
```

(Look at the source code of this post to see the literal formatting, e.g. the use of "batchfile".)